demo/api/package.json

```diff
@@ -20,8 +20,7 @@
         "mikro-orm": "mikro-orm",
         "mikro-orm:drop": "mikro-orm schema:drop -r",
         "mikro-orm:migration:generate": "mikro-orm migration:create",
-        "start": "$npm_execpath prebuild && dotenv -c -- nest start",
-        "start:debug": "$npm_execpath prebuild && dotenv -c -- nest start --debug --watch --preserveWatchOutput",
+        "start": "$npm_execpath prebuild && dotenv -c -- nest start --debug --watch --preserveWatchOutput",
```

Comment:
You'll need to add this to start:dev.

Reply:
Added in [last commit ](https://github.com/vivid-planet/comet/pull/2532/commits/09a4e93d22b7623c5c9e1819853e4b611934dbad)